packages/markdown/component/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@astrojs/markdown",
+  "version": "0.1.0",
+  "type": "module",
```

Comment:
Do we need a `types` field as well for TypeScript support in the Language Server?

Reply:
Yeah, probably, good call.

Reply:
Actually, just adding a `main` field should work.